test/e2e/flowaggregator_test.go

```diff
@@ -300,14 +301,24 @@ func testHelper(t *testing.T, data *TestData, isIPv6 bool) {
 	// and their flow information is exported as IPFIX flow records.
 	// K8s network policies are being tested here.
 	t.Run("IntraNodeFlows", func(t *testing.T) {
-		checkIntraNodeFlows(t, data, podAIPs, podBIPs, isIPv6)
+		label := "IntraNodeFlows"
```

Comment:
can we just use `t.Name()` here, assuming that's a valid label value?

Reply:
No, t.Name() is invalid as it will contain "/" when we are in subtest.